README.md

```diff
@@ -73,7 +74,7 @@ Plug 'SirVer/ultisnips' | Plug 'honza/vim-snippets'
 Plug 'scrooloose/nerdtree', { 'on':  'NERDTreeToggle' }
 Plug 'tpope/vim-fireplace', { 'for': 'clojure' }
 
-" Using git URL
+" Any valid git URL is allowed
 Plug 'https://github.com/junegunn/vim-github-dashboard.git'
```

Comment:
I'm open to a non-Github plugin if anyone has a good suggestion...

@vheon thoughts?